### PR TITLE
GUI: split into multiple files

### PR DESCRIPTION
**Pull request by abhsag24**
makes changing things in the future a lot easier.

Fixes https://github.com/NIT-dgp/UIP/issues/261
Fixes https://github.com/NIT-dgp/UIP/issues/254

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

**Comment by nemaniarjun** [comment]
unack 0ce73f4

**Comment by nemaniarjun** [comment]
just wanted to see all 5 fail :p

**Comment by nkprince007** [comment]
AON, it think its okay here.. 
ack c6c3ca4

**Comment by nkprince007** [comment]
ack c6c3ca4

File: uiplib/gui/generalTab.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+"""Module to create the General Tab in the GUI."""
+
+from tkinter import *
+
+from uiplib.gui.gallery import Gallery
```
Comment (nemaniarjun) [inline uiplib/gui/generalTab.py]: Unneeded new lines ? at `2` and `4`?

File: uiplib/gui/mainGui.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
+"""Module that builds the Graphical User Interface."""
+
+from uiplib.scheduler import scheduler
+from uiplib.setWallpaper import change_background
+from uiplib.utils.utils import update_settings, check_sites
+from uiplib.gui.gallery import Gallery
+from uiplib.gui import generalTab, settingsTab
+
+
```
Comment (nemaniarjun) [inline uiplib/gui/mainGui.py]: Again, unneeded new lines?

**Comment by nkprince007** [comment]
ack c6c3ca4

**Comment by abhsag24** [comment]
merging :D

File: uiplib/gui/mainGui.py (hunk 2 above)
Comment (nemaniarjun) [inline uiplib/gui/mainGui.py]: One new line should be enough.

File: uiplib/gui/settingsTab.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    count_label.grid(row=5, padx=10, pady=10, sticky=W)
+    count.grid(row=5, column=1, sticky=W)
+
+    # Apply
+    apply_button = Button(mainFrame,
+                          text="Apply",
+                          command=lambda: handle_settings(appObj))
+    apply_button.grid(row=6, column=1, pady=20, sticky=W)
+
+
```
Comment (nemaniarjun) [inline uiplib/gui/settingsTab.py]: In other files we have 1 line between methods

File: uiplib/gui/generalTab.py (hunk 1 above)
Comment (abhsag24) [inline uiplib/gui/generalTab.py]: nope seperates library imports and project imports

Comment (nemaniarjun) [inline uiplib/gui/generalTab.py]: at `2` ?

File: uiplib/gui/mainGui.py (hunk 2 above)
Comment (abhsag24) [inline uiplib/gui/mainGui.py]: PEP8 will make a new PR and it'll automatically correct these inconsistencies over all of UIP :P

**Comment by nemaniarjun** [comment]
ack c6c3ca4804c71fc5d5c6af9e7e6e304884c2384f

**Pull request merged**